state/querytracker.go

```diff
@@ -105,7 +105,7 @@ func (q *queryTracker) TrackRead(collectionName string, query interface{}) {
 			Type:           "read",
 			CollectionName: collectionName,
 			Query:          query,
-			Traceback:      string(debug.Stack()),
+			Traceback:      traceback,
```

Comment:
This is funny, we called debug.Stack twice :facepalm:

Reply:
:facepalm: